upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -388,8 +388,8 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 
 	if b.cluster.Spec.Networking.Weave != nil {
 		key := "networking.weave"
-		// 2.1.3-kops.1 = 2.1.3, kops packaging version 1.
-		version := "2.1.3-kops.1"
+		// 2.1.3-kops.2 = 2.1.3, kops packaging version 2.
+		version := "2.1.3-kops.2"
```

Comment:
@chrislovecnm I believe this is the version you mentioned?

Reply:
Sorry if I missed the update ...